src/cljs_test_runner/main.clj

```diff
@@ -74,14 +77,17 @@
     :default "./test"]
    ["-o" "--out PATH" "The output directory for compiled test code"
     :default "./cljs-test-runner-out"]
+   ["-w" "--watch PATH" "Directory to watch for changes (alongside the src-path). May be repeated."
+    :assoc-fn (fn [m k v] (update-in m [k] (fnil conj [:src]) v))]
```

Comment:
What if I _only_ want to watch the source path? If my source of my tests is in `./test` it will mean I'll run `clj -Atest -w test`, this will cause two re-compilations on every change to my test code.

Maybe the default shouldn't be there at all, they should be a distinct set of paths or the default should work differently. So `clj -Atest -w` could watch the source directory but `clj -Atest -w foo` would watch just foo. And `clj-Atest -w foo -w test` would watch the foo and test directories.